modules/swagger-jaxrs2/src/main/java/io/swagger/jaxrs2/OperationParser.java

```diff
@@ -32,256 +35,497 @@
-			return Optional.empty();
-		}
-		Contact contactObject = new Contact();
-		if (StringUtils.isNotBlank(contact.email())) {
-			contactObject.setEmail(contact.email());
-		}
-		if (StringUtils.isNotBlank(contact.name())) {
-			contactObject.setName(contact.name());
-		}
-		if (StringUtils.isNotBlank(contact.url())) {
-			contactObject.setUrl(contact.url());
-		}
-		return Optional.of(contactObject);
-	}
-
-	public static Optional<License> getLicense(io.swagger.oas.annotations.info.License license) {
-		if (license == null) {
-			return Optional.empty();
-		}
-		License licenseObject = new License();
-		if (StringUtils.isNotBlank(license.name())) {
-			licenseObject.setName(license.name());
-		}
-		if (StringUtils.isNotBlank(license.url())) {
-			licenseObject.setUrl(license.url());
-		}
-		return Optional.of(licenseObject);
-	}
+    public static final String RESPONSE_DEFAULT = "default";
+
+    public static Optional<List<Parameter>> getParametersList(io.swagger.oas.annotations.Parameter[] parameters) {
+        if (parameters == null) {
+            return Optional.empty();
+        }
+        List<Parameter> parametersObject = new ArrayList<>();
+        for (io.swagger.oas.annotations.Parameter parameter : parameters) {
+            getParameter(parameter).ifPresent(parametersObject::add);
+
+        }
+        if (parametersObject.size() == 0) {
+            return Optional.empty();
+        }
+        return Optional.of(parametersObject);
+    }
+
+    public static Optional<Parameter> getParameter(io.swagger.oas.annotations.Parameter parameter) {
+        if (parameter == null) {
+            return Optional.empty();
+        }
+        Parameter parameterObject = new Parameter();
+        boolean isEmpty = true;
+        if (StringUtils.isNotBlank(parameter.description())) {
+            parameterObject.setDescription(parameter.description());
+            isEmpty = false;
+        }
+        if (StringUtils.isNotBlank(parameter.name())) {
+            parameterObject.setName(parameter.name());
+            isEmpty = false;
+        }
+        if (StringUtils.isNotBlank(parameter.in())) {
+            parameterObject.setIn(parameter.in());
+            isEmpty = false;
+        }
+        if (parameter.deprecated()) {
+            parameterObject.setDeprecated(parameter.deprecated());
+        }
+        if (parameter.required()) {
+            parameterObject.setRequired(parameter.required());
+            isEmpty = false;
+        }
+        parameterObject.setStyle(StringUtils.isNoneBlank(parameter.style()) ? Parameter.StyleEnum.valueOf(parameter.style()) : null);
+        if (parameter.allowEmptyValue()) {
+            parameterObject.setAllowEmptyValue(parameter.allowEmptyValue());
+            isEmpty = false;
+        }
+        if (parameter.allowReserved()) {
+            parameterObject.setAllowReserved(parameter.allowReserved());
+            isEmpty = false;
+        }
+        if (parameter.explode()) {
```

Comment:
Validation added in the ParameterProcessor, there was functioning correctly here.